crates/meroctl/Cargo.toml

```
@@ -31,6 +31,7 @@ tokio-tungstenite.workspace = true
 url = { workspace = true, features = ["serde"] }
 
 calimero-config.workspace = true
+calimero-context-config = { workspace = true, features = ["client"] }
```

the feature flag shouldn't be necessary

```suggestion
calimero-context-config.workspace = true
```